pages/company/visualizations/company_core_contributors.py

```diff
@@ -18,10 +18,6 @@
 PAGE = "company"
 VIZ_ID = "company-core-contributors"
 
-paramter_1 = "core-contributions-required"
```

Comment:
this is another place that the spelling will be refactored in one shot over codebase,
"paramter" -> "parameter" or "param"

Reply:
This line was removed

Reply:
oh duh, I'm dumb